packages/client/README.md

```diff
@@ -261,6 +266,14 @@ await OpenFeature.close()
 
 ## Extending
 
+### Considerations
```

Comment:
Could you please move this to the bottom on the the extending section?

Reply:
Sure, if you think it's better there.